### ISSUE TITLE
bump cldr to  27.0.1 to fix some incorrect negative prefix pattern for currency?

### ISSUE THREAD

**Issue by ychab** [python-babel/babel#165]
Hi,

If someone got some time to use the latest cldr at http://unicode.org/Public/cldr/27.0.1/core.zip, it should be nice as it may fix some bug.

Indeed, it should fix the french negative prefix '(' deduced from current pattern : 
````
<pattern>#,##0.00 ¤;(#,##0.00 ¤)</pattern>
````
while new ones are more detailled:
````
<currencyFormat type="standard">
  <pattern>#,##0.00 ¤</pattern>
</currencyFormat>
<currencyFormat type="accounting">
  <pattern>#,##0.00 ¤;(#,##0.00 ¤)</pattern>
</currencyFormat>
````
At least we should : 
* change archive relative data at https://github.com/mitsuhiko/babel/blob/master/scripts/download_import_cldr.py#L16
* rewrite https://github.com/mitsuhiko/babel/blob/master/scripts/import_cldr.py#L572 to only use type="standard" (if attribute exists)?
* rerun setup to rebuild locale binary data for packaging
* anything else?

Indeed, the current fix is to set the format manually to use the implicit default minus sign prefix : 
````python
>>> numbers.format_currency(Decimal('-15.45'), 'EUR', format='#,##0.00 ¤', locale='fr')
'-15,45\xa0€'
````
Otherwise, you will got the accounting format (not really usefull in real life!) : 
````python
>>> numbers.format_currency(Decimal('-15.45'), 'EUR', locale='fr')
'(15,45\xa0€)'
````

Thanks !

**Comment by erickwilder** [comment]
@ychab I was looking this very morning about CLDR 27 release notes and I'll make some changes here soon. Thanks for your feedback.

**Comment by etanol** [comment]
Duplicate of #201, and solved by #203

**Comment by etanol** [comment]
Moving the focus to #290
